src/FileHandler.hpp

```diff
@@ -0,0 +1,72 @@
+/**********************************************************************
+* OpenMC2 - An Open Source Re-Implementation of Midnight Club 2
+* Copyright (C) 2016  LRFLEW
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+**********************************************************************/
+
+#pragma once
+
+#include "Addresses.hpp"
+#include <Windows.h>
+
+struct unk_679810_funcTable{
+    HANDLE (*sub_00)(char *path, bool unk2);
+    HANDLE (*sub_04)(char *path);
+    void (*sub_08)();
+    void (*sub_0C)();
+    void (*sub_10)();
+    void (*close_file)(HANDLE file); // 0x14
+    void (*sub_18)();
+    void (*sub_1C)();
+    void (*sub_20)();
+    bool (*sub_24)();
+    void (*sub_28)();
+};
+
+// size: 0x1C
+class FileHandler {
```

Comment:
The more I look at this class, the more I think it's just [FILE](http://www.cplusplus.com/reference/cstdio/FILE/) in the standard library. I'll take a look to see if I can figure out how MSVC defines FILE to compare.